src/const_eval.md

```diff
@@ -57,6 +59,22 @@ A _const context_ is one of the following:
   * [statics]
   * [enum discriminants]
 
+## Const Functions
+
+A _const fn_ is a function that one is permitted to call from a const context. Declaring a function
+`const` has no effect on any existing uses, it only restricts the types that arguments and the
+return type may use, as well as prevent various expressions from being used within it.
+
+Notable features that const contexts have, but const fn haven't are:
+
+* floating point operations
+  * floating point values are treated just like generic parameters without trait bounds beyond
+  `Copy`. So you cannot do anything with them but copy/move them around.
+* `dyn Trait` types
+* generic bounds on generic parameters beyond `Sized`
+* comparing raw pointers
+* union field access or `transmute` invocations.
```

Comment:
```suggestion
* union field access
* `transmute` invocations
```

These two points don't seem to be related to one another, are they?

Also, would you mind linking `transmute` to the library docs?  I think a link to `../core/intrinsics/fn.transmute.html` would be appropriate.

Reply:
well... to me they are, because I was using them for the same thing, but you're right, that view is likely not what anyone else has on it.